pages/developers.tsx

```diff
@@ -8,23 +8,35 @@ import {ApplicationContext} from '../contexts/application';
 import {changeLoadState} from '../contexts/reducers/change-load-state';
 import {IssueData} from '../interfaces/issue-data';
 
-const options_issue = [
+type Filter = {
+  label: string;
+  value: string;
```

Comment:
If you typed the Filter prop, type the possible values as well - because _while they ARE strings_ that string value is being used as an enum